pkg/hubble/relay/server/server.go

```diff
@@ -55,13 +58,20 @@ func New(options ...Option) (*Server, error) {
 		return nil, ErrNoServerTLSConfig
 	}
 
+	var peerSvc peerTypes.ClientBuilder = &peerTypes.LocalClientBuilder{
```

Comment:
nit: the name of the variable is a bit misleading. Could we change it to `peerClientBuilder` or something alike?